### DISCUSSION TITLE
Tests are not running is CI. Is this intentional?

### DISCUSSION THREAD

**Discussion by kishaningithub** [jeroenrinzema/psql-wire#22] [Q&A] [answered]
In [this repos github action](https://github.com/jeroenrinzema/psql-wire/blob/main/.github/workflows/lint.yaml) that runs for every commit and PR i see we are only running `make lint`. 

Is there any reason as to why we are not running `make test` ?

**Comment by jeroenrinzema** [comment, marked as answer]
I did not include the tests inside the CI pipeline. The tests were a bit in-stable. I recently fixed the tests and could safely be included inside the pipeline. Feel free to open a PR to include the tests.

**Reply by kishaningithub** [reply]
Have created a PR :-) https://github.com/jeroenrinzema/psql-wire/pull/23
